### DISCUSSION TITLE
Plex innaccurate movie release (year)

### DISCUSSION THREAD

**Discussion by overtoneblue** [itsToggle/plex_debrid#123] [Ideas]
One issue with using plex watchlist as a base for the query sent to sources, is the release date year is sometimes inaccurate for movies. It's usually only off by a year, however I think a neat feature would be to increment, and decrement the year by 1 and query those years as well, obviously only if this first query using plex's release year returned 0 sources. 

For example, "examplemovie" released in 2002 as listed on plex, however the "real" release date that is picked up by indexers is 2001. If the first query "examplemovie.2002" returns "found 0 releases", send a second query "examplemovie.2001", if this also returns "found 0 releases", query "examplemovie.2003. This could be a scraping option, I wasn't sure if there was an easy way to implement this without touching code of the script itself. I'd love to contribute in that fashion at some point to the project however, what do you think of this workaround for inadequate release dates supplied by plex?

**Comment by itsToggle** [comment]
I had noticed that too, but I was convinced it was only that particular movie (Idiocracy it was I believe). Strange that there seems to be a general problem with Plex being one year off. Your idea sounds great! I'll create a new issue for this and mark it as an enhancement.
